js/src/services/api.service.js

```diff
@@ -27,11 +27,10 @@ class ApiService {
    * will work just fine, even if $ is not assign globally.
    *
    * @param code // javascript to be eval.
-   * @param jQueryRef // reference to jQuery.
+   * @param $ // reference to jQuery.
    */
-    evalResponse(code, jQueryRef) {
-        const $ = jQueryRef; //eslint-disable-line
-        window.eval(code);
+    evalResponse(code, $) { //eslint-disable-line
```

Comment:
would be good to add fixer for comments, to always normalize to `//<SP>text` and `/*<SP>text<SP>*/`

Reply:
I added spaced-comment rule. It will at least catch them next time.